plugins/org.locationtech.udig.tool.edit.tests/src/org/locationtech/udig/tools/edit/BehaviourCommandTest.java

```diff
@@ -90,6 +125,8 @@ public Map getMap() {
                 public void run( IProgressMonitor monitor ) throws Exception {
                     if( ran )
                         fail();
+                    Thread.sleep(10);
```

Comment:
IMO checkes uses a Timestamp rather than a [CountDownLatch](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CountDownLatch.html). 
Otherwise more and more testcases would increase build / test runtime ..